### PR TITLE
Added KubeMQ message broker

### PR DESCRIPTION
**Pull request by Ido-Sheffer**

### CODE REVIEW

**Comment by guangbochen** [comment]
@Ido-Sheffer can u please help to rebase the commit against the `rancher:partners` branch, thanks.

File: charts/kubemq/v0.1.0/questions.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+labels:
+  io.cattle.role: project
```
Comment (guangbochen) [inline charts/kubemq/v0.1.0/questions.yml]: Can you please add the partner label, thanks
```
  io.rancher.certified: partner
```

**Comment by deniseschannon** [comment]
@guangbochen is everything okay besides the partner label?

**Comment by guangbochen** [comment]
@deniseschannon user needs to rebase the commit and add the partner label.

File: charts/kubemq/v0.1.0/questions.yml (hunk 1 above)
Comment (Ido-Sheffer) [reply charts/kubemq/v0.1.0/questions.yml]: O.K. Done.
Thanks

Comment (Ido-Sheffer) [reply charts/kubemq/v0.1.0/questions.yml]: Made a new pull request: Added KubeMQ message broker. #329

**Pull request closed without merging**